src/components/BitcoinAmountInput.tsx

```diff
@@ -114,9 +114,9 @@ const BitcoinAmountInput = forwardRef(
               field.onBlur(e)
             }}
             onChange={(e) => {
-              const valueOrNan = parseFloat(e.target.value ?? '')
-
-              if (!isValidNumber(valueOrNan)) {
+              const valueOrNan = e.target.value ?? ''
```

Comment:
Before, the var named `valueOrNan` was a "value" (number) or "NaN".. now it is the raw user input or empty, so the name should change as well, otherwise it is quite surprising.

Reply:
Renamed valueOrNan to rawUserInputOrEmpty for clarity